### PR TITLE
Vaes_et_al. APWP

### PR DESCRIPTION
**Pull request by dpastorgalan**
I have added the full APWP of Bram Vaes. In my dummy version of pmag.org where I play with new add-ons it works fine. Douwe, Bram and I have tested it several weeks already. I am submitting this to the DEV version now so you can check it. Douwe wants this change soon, so if you think it looks fine, move it to master ASAP (A lot more to come in the next months, but with a lesser rush).

### CODE REVIEW

File: geography/index.html [hunk 1]
```diff
@@ -92,8 +92,8 @@
                <br>
 
                <div class="alert alert-light" role="alert">
-                  <h5> Paleomagnetism.org 2 - Geography </h5>
```
Comment (Jollyfant) [inline geography/index.html:95]: Is this change intended?

File: geography/index.html [hunk 2]
```diff
@@ -165,7 +165,7 @@ <h5> Paleomagnetism.org 2 - Magnetostratigraphy </h5>
                <br>
                <div class="alert alert-light alert-dismissible fade show" role="alert">
                   <h5> Geography - Map Overview & APWPs </h5>
-                  <p> This map overview illustrates vertical-axis rotations for the selected sites including confidence parachutes (ΔDx). Declinations, inclinations, paleolatitudes, and apparant polar wander paths can be plotted for major plates. <p> <b>Tip!</b> You can create your own APWPs using <a href='https://apwp-online.org'>APWP-Online</a> by Vaes et al., 2023.
+                                   <p> This map overview illustrates vertical-axis rotations for the selected sites including confidence parachutes (ΔDx). Declinations, inclinations, paleolatitudes, and apparant polar wander paths can be plotted for major plates. <p> <b>Tip!</b> You can create your own APWPs using <a href='https://apwp-online.org'>APWP-Online</a> by Vaes et al., 2023.
```
Comment (Jollyfant) [inline geography/index.html:168]: Same here

**Review by Jollyfant — COMMENTED** [review]
What was changed in apwp.json? It's a big file with a lot of changes and it is hard to see. I think vaes, 2022 was already in there. Correct?

File: geography/index.html (hunk 2 above)
Comment (dpastorgalan) [reply geography/index.html:168]: Hi!
The previous path needed a rotation file to work because the plate circuit was incomplete. I have added Bram's full plate circuit at the end of the database (to keep the previous chronological order). The two changes you mention, I remember that there was some conflict between dev version and master versions (both have some commits ahead of the other) so I decided to use one of them (I guess the master?) and copy some stuff from one to other. Also I have corrected Vaes et al., 2022 for 2023 where it was 2022 (it was finally published in 2023). I have tested it and it works fine in my dummy version (dpastorgalan.github.io).

A thing that would be easier for next changes is if we make sure that the only one that is ahead is dev. (?)

Comment (Jollyfant) [reply geography/index.html:168]: Always branch from dev and commit to dev and there should be no conflicts. Git can be a bit tricky though. Can you commit a change that fixes the problems I highligted? It changed the text from geography to statistics in the geography portal.

Comment (dpastorgalan) [reply geography/index.html]: I think I have correct this properly. Now it is geography again. I have also corrected the apparant for apparent typo. Since I had a mess, I have decided to use my own dev branch forked from yours to submit all other changes that I have in my playground branches. So I can go one by one and avoid messing around again. Cheers!

**Comment by Jollyfant** [comment]
I rather close this pull request and merge a new one. Just clone this repository, checkout the `dev` branch, make your changes to it and commit to `dev`. Then make a pull request from `dev` -> `dev` and it will work. Also don't forget to update the changelog!

**Comment by Jollyfant** [comment]
Actually I decided to merge it. Your version should be on beta.paleomagnetism.org now.

**Comment by dpastorgalan** [comment]
Ok, got it. I am resetting my own forked version. From now on I will submit from my dev to the official dev only. Cheers and sorry for the mess!

**Comment by Jollyfant** [comment]
No worries, let me know if everyhing is ok on beta. Then I will merge this new pull request from `dev` to `master` and version 2.5.0 will go live.

https://github.com/Jollyfant/PMAG2/pull/88
